### PR TITLE
Gidk 27by7a heading prod

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

**Comment by shalinee-walkover** [comment]
https://app.clickup.com/t/27by7a
Developer - @ravinderwalkover @shalinee-walkover

Description - Heading issue resolved, Heading should be visible on small screen

File: apps/web-giddh/src/app/invoice/invoice.component.ts [hunk 1]
```diff
@@ -107,6 +111,60 @@ export class InvoiceComponent implements OnInit, OnDestroy, AfterViewInit {
...
+                case 'invoice':
+                    pageHeading = this.localeData?.tabs?.invoices;
+                    break;
+                case 'sales':
+                    pageHeading = this.localeData?.tabs?.invoices;
+                    break;
+                case 'recurring':
+                    pageHeading = this.localeData?.tabs?.recurring;
+                    break;
+                case 'pending':
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/invoice/invoice.component.ts:155]: case 'pending', 'templates', 'settings' are redundant from line 155

File: apps/web-giddh/src/app/settings/profile/setting.profile.component.ts [hunk 2]
```diff
@@ -57,6 +58,8 @@ export interface IGstObj {
     ]
 })
 export class SettingProfileComponent implements OnInit, OnDestroy {
+    @Output() public pageHeading: EventEmitter<string> = new EventEmitter();
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/settings/profile/setting.profile.component.ts:61]: MIssing JS doc

File: apps/web-giddh/src/app/settings/settings.component.ts [hunk 3]
```diff
@@ -39,6 +39,8 @@ export class SettingsComponent implements OnInit, OnDestroy {
     @ViewChild('permissionComp', { static: false }) public permissionComp: SettingPermissionComponent;
     @ViewChild('tagComp', { static: false }) public tagComp: SettingsTagsComponent;
 
+
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/settings/settings.component.ts:42]: Avoid empty lines

File: apps/web-giddh/src/app/tb-pl-bs/tb-pl-bs.component.ts [hunk 4]
```diff
@@ -41,15 +44,46 @@ export class TbPlBsComponent implements OnInit, OnDestroy {
     /* This will hold common JSON data */
     public commonLocaleData: any = {};
 
-    constructor(private store: Store<AppState>, private companyActions: CompanyActions, private _route: ActivatedRoute, private router: Router, private breakPointObservar: BreakpointObserver,) {
+    constructor(private store: Store<AppState>, private companyActions: CompanyActions, private _route: ActivatedRoute, private router: Router, private _generalActions: GeneralActions, private breakPointObservar: BreakpointObserver,) {
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/tb-pl-bs/tb-pl-bs.component.ts:47]: Avoid introduction of unused dependencies `_generalActions`

File: apps/web-giddh/src/app/tb-pl-bs/tb-pl-bs.component.ts [hunk 5]
```diff
@@ -7,6 +7,7 @@ import { CompanyResponse, StateDetailsRequest } from '../models/api-models/Compa
 import { CompanyActions } from '../actions/company.actions';
 import { ReplaySubject } from 'rxjs';
 import { TabsetComponent } from 'ngx-bootstrap/tabs';
+import { GeneralActions } from '../actions/general/general.actions';
```
Comment (prateek-giddh) [inline apps/web-giddh/src/app/tb-pl-bs/tb-pl-bs.component.ts:10]: Remove unused import

**Review by prateek-giddh — CHANGES_REQUESTED** [review]
Please check all the comments on lower env.

File: apps/web-giddh/src/app/invoice/invoice.component.ts (hunk 1 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/invoice/invoice.component.ts:155]: Done

File: apps/web-giddh/src/app/tb-pl-bs/tb-pl-bs.component.ts (hunk 4 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/tb-pl-bs/tb-pl-bs.component.ts:47]: Done

File: apps/web-giddh/src/app/settings/settings.component.ts (hunk 3 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/settings/settings.component.ts:42]: Done

File: apps/web-giddh/src/app/settings/profile/setting.profile.component.ts (hunk 2 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/settings/profile/setting.profile.component.ts:61]: Done

File: apps/web-giddh/src/app/tb-pl-bs/tb-pl-bs.component.ts (hunk 5 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/app/tb-pl-bs/tb-pl-bs.component.ts:10]: Done

**Review by prateek-giddh — APPROVED** [review]

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**